### PR TITLE
`Exam mode`: Fix tooltip for test exams

### PR DESCRIPTION
**Pull request by mallgaier**
#### General
- [ ] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [ ] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/) and ensured that the layout is responsive.
- [x] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.

### Motivation and Context
The tooltip for the exam mode within the exam update component currently descripes, that test exams could be conducted multiple times by the students.
As this feature is not yet implemented, the tooltip has to be updated.

### Description
The textual description for the tooltip has been updated to highlight, that students can start the test exam at any time within the working window, but are not able to conduct a test exam multiple times

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to Course Administration
3. Create an Exam
4. Hover over the tooltip for the Exam Mode Selection and make shure, that the text for the tooltip is displayed accordingly. 

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2

### Screenshots
image

### CODE REVIEW

File: src/main/webapp/i18n/de/exam.json [hunk 1]
```diff
@@ -377,7 +377,7 @@
                 "realExam": "Klausur",
                 "testExam": "Testklausur",
                 "examMode": "Exammodus",
-                "examModeTooltip": "Testklausuren können von Studierenden mehrmals innerhalb des Bearbeitungsfensters durchgeführt werden und erhalten nach jedem Durchlauf automatisiertes Feedback. HINWEIS: Der Modus kann nach der Erstellung nicht mehr geändert werden!",
+                "examModeTooltip": "Testklausuren können von Studierenden einmal frei innerhalb des Bearbeitungsfensters begonnen werden, zur Durchführung steht die gewählte Arbeitszeit zur Verfügung. Nach dem Durchlauf erhalten die Studierenden automatisiertes Feedback. HINWEIS: Der Modus kann nach der Erstellung nicht mehr geändert werden!",
```
Comment (Strohgelaender) [inline src/main/webapp/i18n/de/exam.json:380]:
```suggestion
                "examModeTooltip": "Testklausuren können von Studierenden an einem frei wählbaren Zeitpunkt innerhalb des Bearbeitungsfensters begonnen werden, zur Durchführung steht die gewählte Arbeitszeit zur Verfügung. Nach dem Durchlauf erhalten die Studierenden automatisiertes Feedback. HINWEIS: Der Modus kann nach der Erstellung nicht mehr geändert werden!",
```

**Review by TheZoker — DISMISSED** [review]
Apart from the already mentioned comment, it looks good to me!

File: src/main/webapp/i18n/de/exam.json (hunk 1 above)
Comment (mallgaier) [reply src/main/webapp/i18n/de/exam.json]: 👍

**Review by b-fein — APPROVED** [review]
Thanks!

**Review by simon-hng — APPROVED** [review]

**Review by Strohgelaender — APPROVED** [review]
Thanks for directly adressing this.

**Pull request merged**